rapids-cmake/export/write_dependencies.cmake

```diff
@@ -88,14 +89,12 @@ endif()\n")
     endif()
   endif()
 
-  if(find_root_dirs)
-    foreach(package IN LISTS find_root_dirs)
-      get_property(root_dir_path TARGET rapids_export_${type}_${export_set}
-                   PROPERTY "FIND_ROOT_FOR_${package}")
-      set(dep_content "set(${package}_ROOT \"${root_dir_path}\")")
-      string(APPEND _RAPIDS_EXPORT_CONTENTS "${dep_content}\n")
-    endforeach()
-  endif()
+  foreach(package IN LISTS find_root_dirs)
```

Comment:
What changed that `find_root_dirs` is guaranteed to be defined?

Reply:
`foreach( IN LISTS)` will ignore empty variable, so the if condition was never needed. Just noticed now that I could remove it.